raiden/utils/__init__.py

```diff
@@ -190,8 +190,12 @@ def wait_until(func, wait_for=None, sleep_for=0.5):
         func(): result of func, if truth value, or None"""
     start = time.time()
     res = func()
-    while (True if wait_for is None else (time.time() - start < wait_for))\
-            and not res:
+    loop_cond = True
+    while loop_cond:
         gevent.sleep(sleep_for)
         res = func()
+        loop_cond = (
+            (True if wait_for is None else (time.time() - start < wait_for)) and
```

Comment:
Well, my point with no using multi line if is readability ... this doesn't help much IMO, suggestion:

```python
stop = func()

if stop:
  return

if wait_for:
  deadline = time.time() + wait_for
  while not stop and time.time() <= deadline:
    gevent.sleep(sleep_for)
    stop = func()

else:
  while not stop:
    gevent.sleep(sleep_for)
    stop = func()
```

Reply:
I agree that multi-line conditions should be avoided for readability reasons, but I don't think that being so verbose as your suggestion is really needed.

But since you requested this change I will do it as I don't have a preference of one over the other.